batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/dao/EmployeeDAOImpl.java

```diff
@@ -81,8 +85,9 @@ public void create(Employee employee) throws DataException {
     }
 
     public void edit(Employee employee) throws DataException {
-        try (Connection conn = DBConnection.getConnection(); PreparedStatement ps = conn.prepareStatement(SqlQueryConstants.UPDATE_QUERY)) { // NOSONAR         
-            setQueryAttribute(ps, employee).setInt(5, employee.getId());
+        try (Connection conn = DBConnection.getConnection(); PreparedStatement ps = conn.prepareStatement(UPDATE_QUERY)) {
+            setQueryAttribute(ps, employee).setInt(6, employee.getId());
+            System.out.println(ps);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)